op-scim-bridge/chart/op-scim-bridge/templates/op-scim-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}-op-scim-bridge
+  labels: &ScimDeploymentLabels
+    app.kubernetes.io/name: {{ .Values.name }}
+    app.kubernetes.io/component: op-scim-bridge
+spec:
+  selector:
+    matchLabels: *ScimDeploymentLabels
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels: *ScimDeploymentLabels
+    spec:
+      initContainers:
+      - name: scimuser-home-permissions
+        image: alpine:3.13.1
+        command: ["/bin/sh", "-c"]
+        args: ["mkdir -p {{ .Values.scimUserHome }} && chown -R {{ .Values.scimUserID }} {{ .Values.scimUserHome }}"]
+        volumeMounts:
+        - mountPath: "/home"
+          name: {{ .Values.name }}-scimsession
+      containers:
+      - name: op-scim-bridge
+        image: {{ .Values.bridge.image.repository }}:{{ .Values.bridge.image.tag }}
+        imagePullPolicy: Always
+        command: ["/op-scim/op-scim"]
+        env:
+        - name: "OP_PORT"
+          value: "8080"
+        - name: "OP_SESSION"
+          value: "{{ .Values.scimUserHome }}/{{ .Values.scimsessionFile }}"
+        - name: "OP_REDIS_HOST"
+          value: "{{ .Values.name }}-op-scim-redis-svc"
+        - name: "OP_ONE_CLICK"
+          value: "true"
```

Comment:
Same question as above: why do we no longer set the `OP_DOMAIN` value from `OP_ACCOUNT_DOMAIN`?

In the old version it was:

```
 - name: OP_DOMAIN
          value: {{ .Values.OP_ACCOUNT_DOMAIN }}
 ```